powerfactory_utils/exporter/exporter.py

```diff
@@ -1374,8 +1373,8 @@ def create_q_controller(
                 elif controller_type == ControllerType.Q_U:
                     u_nom = round(ext_ctrl.refbar.uknom, DecimalDigits.VOLTAGE) * Exponents.VOLTAGE  # voltage in V
 
-                    qmax_ue = abs(ext_ctrl.Qmin / p_r)  # per unit
-                    qmax_oe = abs(ext_ctrl.Qmax / p_r)  # per unit
+                    qmax_ue = abs(ext_ctrl.Qmin)
```

Comment:
```suggestion
                    qmax_ue = round(abs(ext_ctrl.Qmin), DecimalDigits.POWER)
```

Reply:
decline. rounding is intended at the end of the function, right?
q_r is oinly used for qmax and q_max is rounded at the end.